sdk/opendp/whitenoise/client/__init__.py

```diff
@@ -45,10 +57,11 @@ def register(self, dataset):
             dataset_type=dataset['dataset_type'], \
             budget=dataset['budget'], \
             csv_details=dataset['csv_details'], \
-            dataverse_details=dataset['dataverse_details'])
+            dataverse_details=dataset['dataverse_details'], \
+            released=False)
         return client.datasetregister(register_request)
 
-    def read(self, dataset_name, budget):
+    def read(self, dataset_name, budget=None):
```

Comment:
I would prefer a read_release than have budget be optional. 

These are very distict user actions so we should keep them separate. One is used by the analyst the other  is used by the module. Over time the credentials needed for both actions will also be different